TKeazirian.HTTPServer/Server.cs

```diff
@@ -4,13 +4,13 @@
 
 namespace TKeazirian.HTTPServer;
 
-public static class EchoServer
+public static class Server
 {
     private static string? _request;
 
     public static void StartListening()
     {
-        var ipAddress = IPAddress.Any;
+        var ipAddress = IPAddress.Parse("127.0.0.1");
```

Comment:
anytime I see numbers hardcoded into code like this, I always ask can this be extracted to something better? (variable, parameter, constant, config, etc ...). Its the "magic number" code smell.

Reply:
Makes sense!  I can extract it and put it into a variable top level.